tools/generator/AnDOGenerator.py

```diff
@@ -123,14 +123,14 @@ def __init__(self, expName=None, guid=None, sesID=None, date=None, sesNumber=Non
     def get_session_path(self):
         path = os.path.join(f'exp-{self.expName}',
                             f'sub-{self.guid}',
-                            f'ses-{self.sesID}')
+                            f'ses-{self.sesID}',
+                            f'ephys')
```

Comment:
I am not sure that this would be the right place for this change, as this is class is still intended for folder handling on the session level. Maybe revert this change here and just replace the datafolder list below for now?

Reply:
OK, reverted and replacing the list below.